run-dev.sh

```diff
@@ -0,0 +1,6 @@
+# Runs both dev servers at the same time, can be ended at the same time with ctrl+C
+echo "Preview on http://localhost:9000"
+
+trap "kill %1" SIGINT
+(cd frontend/; npm run start-dev) | tee frontend.log | sed -e "s/^/\x1b[32m[FRONTEND]\x1b[0m /" \
```

Comment:
If they're already going to different logfiles, why do the lines need to be prefixed?

Reply:
The colored prefix helps for distinguishing the different sources in the terminal when they're running.  Especially when things are scrolling by fast, it's a nice way to visually distinguish what's happening (hopefully this image shows what I mean).
![image](https://user-images.githubusercontent.com/65669542/87444300-3e6bd800-c5bc-11ea-9e4c-fb8335108963.png)

Reply:
Ohh, for the STDOUT log. Gotcha yeah that makes sense